packages/opencensus-exporter-zpages/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@opencensus/exporter-zpages",
+  "version": "0.0.1",
+  "description": "Package Zpages implements a collection of HTML pages that display stats and trace data",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "census-instrumentation/opencensus-node",
+  "scripts": {
+    "test": "nyc -x '**/test/**' --reporter=html --reporter=text mocha 'build/test/**/*.js'",
+    "clean": "rimraf build/*",
+    "check": "gts check",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile; npm run cp_templates",
+    "pretest": "npm run compile; npm run cp_templates",
+    "posttest": "npm run check",
+    "cp_templates": "cp -r ./src/zpages-frontend/templates ./build/src/zpages-frontend/"
+  },
+  "keywords": [
+    "opencensus",
+    "nodejs",
+    "tracing",
+    "profiling"
+  ],
+  "author": "Google Inc.",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "CHANGELOG.md",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@types/express": "^4.11.1",
+    "@types/extend": "^3.0.0",
+    "@types/mocha": "^2.2.48",
+    "@types/node": "^9.4.7",    
+    "gts": "^0.5.4",
+    "mocha": "^5.0.4",
+    "nyc": "11.6.0",
+    "rimraf": "^2.6.2",     
+    "ts-node": "^4.0.0",
+    "typescript": "~2.6.1"
+  },
+  "dependencies": {
+    "@opencensus/opencensus-core": "^0.0.1",
+    "@opencensus/nodejs": "^0.0.1",
+    "ejs": "^2.5.8",
+    "express": "^4.16.3",
+    "time-unit": "^1.1.0"
```

Comment:
Not to discredit the authors of these modules, but I am always wary of introducing a dependency on a seldom-used module. The functionality of this module seems straightforward enough for us to implement it ourselves (I'm not sure why we need the class to be Java-like). Can we remove this dependency?

Reply:
Ok. Removed.